checkov/kustomize/runner.py

```diff
@@ -232,23 +261,32 @@ def run(self, root_folder, external_checks_dir=None, files=None, runner_filter=R
                         self.kustomizeProcessedFolderAndMeta[filePath]['overlay_name'] = checkovKustomizeEnvNameByPath
                         logging.warning(f"Could not confirm base dir for Kustomize overlay/env. Using {checkovKustomizeEnvNameByPath} for Checkov Results.")
             
+
+                if self.templateRendererCommand == "kubectl":
+                    templateRenderCommandOptions = "kustomize"
+                if self.templateRendererCommand == "kustomize":
```

Comment:
```suggestion
                elif self.templateRendererCommand == "kustomize":
```
otherwise both `if` checks will be checked each time